frontend/src/contexts/HostContext/HostReducer.js

```diff
@@ -212,7 +218,11 @@ function kickPlayer(state, { playerId }) {
 
   if (
     removingCzar &&
-    newPlayerIds.filter((id) => state.players[id].isConnected).length > 1 &&
+    newPlayerIds.filter(
+      (id) =>
+        state.players[id].status === 'playing' ||
+        state.players[id].status === 'staging',
```

Comment:
We could use a .includes here to clean this up:

`['playing', 'staging'].includes(state.players[id].status`